close #23 navbar

AbdelrahmanEmadEwis opened the pull request:
1) 3 buttons with hover transitions and click transitions. 
2) linear gradient background.
3) logo with "SickMan" title  and a syringe icon.
4) both bottom border and left border are solid, white and circular.

AbdelrahmanEmadEwis committed:
close #23 navbar

1) 3 buttons with hover transitions and click transitions
2) linear gradient background
3) logo with "sickman" title  and a syringe icon
4) both bottom border and left border are solid, white and circular

DevAbdoTolba commented on front/src/components/AppBar.tsx:
```diff
@@ -1,7 +1,121 @@
+      // , "&::after": {
+      //   content: '""', position: "absolute", width: "100%", height: "100%", borderRadius: "15rem"
+      //   , border: "1px solid black", p: "5px", top: "0"
+      // },
```
remove redundant comments

DevAbdoTolba commented on front/src/components/AppBar.tsx:
```diff
@@ -1,7 +1,121 @@
+
+
+
+
```
remove extra white-spaces

DevAbdoTolba commented on front/src/components/AppBar.tsx:
```diff
@@ -1,7 +1,121 @@
+import { Stack, Button, Typography, Box, Paper } from '@mui/material';
+import { Block } from '@mui/icons-material';
+import VaccinesIcon from '@mui/icons-material/Vaccines';
+import { after } from 'node:test';
```
remove unused  `import { after } from 'node:test';`

DevAbdoTolba reviewed (changes requested):
Great!! 🫡🫡. Small changes required

AbdelrahmanEmadEwis committed:
Applied small changes:-

1)removed redundant comments

2)remove extra white-spaces

3)removed unused import { after } from 'node:test';

DevAbdoTolba reviewed (approved):
LGTM 👍
